src/_includes/cwl/hello-world-expressionlib.cwl

```diff
@@ -0,0 +1,25 @@
+cwlVersion: v1.2
+class: CommandLineTool
+requirements:
+  - class: InlineJavascriptRequirement
+    expressionLib:
+      - { $include: custom-functions.js }
+      - |
+        /**
+         * An example function that uses a function from the included
+         * custom-functions.js file to create a Hello World message.
+         * @param {Object} message - CWL document input message
+         */
+        var createHelloWorldMessage = function (message) {
+          return capitalizeWords(message);
+        };
```

Comment:
Some readers might interpret that all `$include`d functions need a wrapping function here in `expressionLib`.

I think it would be better to start with `capitalizeWords` defined inline in `expressionLib` and then show how  `$include: custom-functions.js` allows centralizing those functions.

As a 3rd example you could show a more meaningful adaptation of the external JS function inside `expressionLib`.

Reply:
Latest commit should have a new version of the text where I tried to explain it in parts to the user as you suggested @mr-c. :wave: Thanks!!